Packs/Base/Scripts/DBotFindSimilarIncidents/DBotFindSimilarIncidents.py

```diff
@@ -0,0 +1,854 @@
+    return incident, incident_id
+
+
+def remove_fields_not_in_incident(*args, incorrect_fields):
+    """
+    Return list without field in incorrect_fields
+    :param args: *List of fields
+    :param incorrect_fields: fields that we don't want
+    :return:
+    """
+    return [[x for x in field_type if x not in incorrect_fields] for field_type in args]
+
+
+def get_similar_incidents_by_indicators(args: Dict):
+    """
+    Use DBotFindSimilarIncidentsByIndicators automation and return similars incident from the automation
+    :param args: argument for DBotFindSimilarIncidentsByIndicators automation
+    :return:  return similars incident from the automation
+    """
+    res = demisto.executeCommand('DBotFindSimilarIncidentsByIndicators', args)
+    if is_error(res):
+        return_error(get_error(res))
+    return res[1]['Contents']
+
+
+def dumps_json_field_in_incident(incident: Dict):
+    """
+    Dumps value that are dict in for incident values
+    :param incident: json representing the incident
+    :return:
+    """
+    for field in incident.keys():
+        if isinstance(incident[field], dict):
+            incident[field] = json.dumps(incident[field])
+    incident_df = pd.DataFrame.from_dict(incident, orient='index').T
+    return incident_df
+
+
+def return_outputs_summary(confidence: float, number_incident_fetched: int, number_incidents_found: int,
+                           fields_used: List[str], global_msg: str) -> None:
+    """
+    Return entry for summary of the automation - Give information about the automation run
+    :param confidence: confidence level given by the user
+    :param number_incident_fetched: number of incident fetched from the instance
+    :param number_incidents_found: number of similar incident found
+    :param fields_used: Fields used to find similarity
+    :param global_msg: informative message
+    :return:
+    """
+    summary = {
+        'Confidence': str(confidence),
+        'Incident fetched with exact match': number_incident_fetched,
+        'Number of similar incident found ': number_incidents_found,
+        'Fields used for similarity (not empty)': ', '.join(fields_used),
+        'Additional message': global_msg
+    }
+    return_outputs(readable_output=tableToMarkdown("Summary", summary))
+
+
+def create_context_for_incidents(similar_incidents=pd.DataFrame()):
+    """
+    Return context from dataframe of incident
+    :param similar_incidents: DataFrame of incidents with indicators
+    :return: context
+    """
+    if len(similar_incidents) == 0:
+        context = {
+            'similarIncidentList': {},
+            'isSimilarIncidentFound': False
+        }
+    else:
+        context = {
+            'similarIncident': (similar_incidents.to_dict(orient='records')),
+            'isSimilarIncidentFound': True
+        }
+    return context
+
+
+def return_outputs_similar_incidents(show_actual_incident: bool, current_incident: pd.DataFrame,
```

Comment:
1. format the `created` field, you return a raw timestamp (check `timestamp_to_datestring` at `commonserverypython`)
2. keep same headers order and capitalization between current incident table to similar incidents tables 
3. Change `Actual incident` header to "Current Incident" or "Input incident", actual is not clear

Reply:
Hi please notice also 2: replace the names of constant fields (id, name, type, created) with the right capitalization as same as they appear in the incidents table, and when `showActualIncident==True`, return headers in the same order they appeared in the incidents table